p/themes/base-theme/frss.css

```diff
@@ -501,7 +514,20 @@ td.numeric {
 	}
 }
 
+input#favicon-upload {
+	display: none;
+}
+
 /*=== Buttons */
+button[disabled] {
+	opacity: 0.5;
+	color: #a6a6a6;
+}
```

Comment:
could you please have a look onto `input:disabled`? It would be great to have it very similar/the same.

Regarding the color: please use variables for colors. Please no plain color anymore. It would be very useful for the other themes or dark mode or user theme.